lib/charms/alertmanager_k8s/v0/alertmanager_dispatch.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""# Alertmanager library.
+
+This library is designed to be used by a charm consuming or providing the `alerting` relation.
```

Comment:
Is that "alerting" relation or "alerting" interface. Libraries define the behaviour of relation interfaces. The relation names themselves are arbitrary.

Reply:
@sed-i seems the Alertmanager library docstring is outdated.

Reply:
Originally I was indeed referring to the relation name. [Changed](https://github.com/canonical/alertmanager-operator/pull/34/commits/58e733fafa1ccf2ec9e0ba3406f57035cfccb2c5) to interface name.